doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -806,12 +818,12 @@ a query or geographical query). A given entity have to match all the criteria to
 See [Simple Query Language](#simple_query_language) and [Geographical Queries](#geographical_queries). 
 
 The response payload is an Array which contains one object per matching entity. Each entity follows
-the JSON entity representation format (described in a section above).
+the JSON entity representation format (described in "JSON Entity Representation" section).
 
 Response code:
 
 * Successful operation uses 200 OK
-* Errors use a non-200 and (optionally) an error payload. See subsection on Error Responses for more details.
+* Errors use a non-200 and (optionally) an error payload. See subsection on "Error Responses" for more details.
```

Comment:
perhaps we could say errors use a 40x or 50x as non 200, for instance 201 , mean success as well.